sdks/python/apache_beam/io/external/snowflake.py

```diff
@@ -0,0 +1,185 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# pytype: skip-file
+
+from __future__ import absolute_import
+
+import typing
+
+from past.builtins import unicode
+
+import apache_beam as beam
+from apache_beam.transforms.external import BeamJarExpansionService
+from apache_beam.transforms.external import ExternalTransform
+from apache_beam.transforms.external import NamedTupleBasedPayloadBuilder
+
+"""Snowflake transforms tested against Flink portable runner.
+  **Setup**
+  Transforms provided in this module are cross-language transforms
+  implemented in the Beam Java SDK. During the pipeline construction, Python SDK
+  will connect to a Java expansion service to expand these transforms.
+  To facilitate this, a small amount of setup is needed before using these
+  transforms in a Beam Python pipeline.
+  There are several ways to setup cross-language Snowflake transforms.
+  * Option 1: use the default expansion service
+  * Option 2: specify a custom expansion service
+  See below for details regarding each of these options.
+  *Option 1: Use the default expansion service*
+  This is the recommended and easiest setup option for using Python Kafka
+  transforms. This option is only available for Beam 2.22.0 and later.
```

Comment:
Pls drop "This option is only available for Beam 2.22.0 and later". This was just for Kafka.

Reply:
Thanks, I fixed it.